### ISSUE TITLE
`image_modifier` is applied only to DL1ab

### ISSUE THREAD

**Issue by vuillaut** [cta-observatory/cta-lstchain#1037]
I don't think that should be the case.
Whatever tuning is in the config should be applied during the process, also `r0_to_dl1`.

**Comment by maxnoe** [comment]
Isn't the idea that the initial r0 to dl1 contains the unmodified images and then we use dl1ab with the pedestal information available to refine that?

**Comment by moralejo** [comment]
Yes, that is the idea. I also dislike having no version of DL1-MC which  contains no randomness. It makes searching for issues much harder. So I would simply make clear somehow that the simple version of the NSB tuning (i.e. the post-calibration one) is not applicable in the R0 to DL1 stage.

**Comment by rlopezcoto** [comment]
what about calling it `image_modifier_dl1ab` (or something like that)?

**Comment by vuillaut** [comment]
Ok that's an alternative solution, but at the moment I find it very confusing (and a clear source of error).

**Comment by maxnoe** [comment]
I think it is more confusing to use the same configuration files for all tools and not raise errors for unknown configuration options....
